### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by frennkie**
Some little updates for the tests. Still working on viper.common.objects.File.

### CODE REVIEW

File: viper/common/objects.py [hunk 1]
```diff
@@ -105,11 +105,7 @@ def __init__(self, path):
         self.children = ''
 
         if self.is_valid():
-            if sys.version_info < (3, 0):
-                # PY2SUPPORT
-                self.name = os.path.basename(self.path).encode('utf-8')
-            else:
-                self.name = os.path.basename(self.path)
+            self.name = os.path.basename(self.path)
```
Comment (frennkie) [inline viper/common/objects.py:108]: @Rafiot could you please have a look at this. I ran unittest on Py35 and Py27 and the Py2 tests fail on the ".encode('utf-8')" command. Removing it fixes this and the tests work for both 2+3. So it seems this differentiation is not needed - or there is something wrong with the Unitttests... :-)

Comment (Rafiot) [reply viper/common/objects.py:108]: I can't remember why I did it, but if it works now, much better :)

**Comment by frennkie** [comment]
I did some more testing... Having a file with non ASCII only file name in Py2 Viper cause some more crashes.. (e.g. open -l). But I think the changes here in this PR are still correct .. but some more changes would be needed to Support that scenario (which we don't really have to.. in my opinion).

I'll look into it later tonight.

**Comment by Rafiot** [comment]
If something is broken for a long time in python2 but works in python3, I'd only fix it if it doesn't require too much time/code

**Pull request merged**